src/js/components/Meter/index.d.ts

```diff
@@ -19,7 +19,7 @@ export interface MeterProps {
   round?: boolean;
   size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'full' | string;
   thickness?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string;
-  type?: 'bar' | 'circle' | 'pie';
+  type?: 'bar' | 'circle' | 'pie' | 'semicircle';
```

Comment:
@jcfilben Out of interest, is there a reason this is documented in the official docs but wasnt included in `index.d.ts` files until now?

Reply:
@alexmirrington thank you for catching this! It should of been included.